### ISSUE TITLE
Clicking "subreddit moderators" on sidebar of /r/reddit.com causes app to hang (kinda)

### ISSUE THREAD

**Issue by incIined** [ccrama/Slide#2562]
The screen won't respond but I can still swipe out of the subreddit and the app will work fine. 

Also clicking the suggestion of /r/reddit.com in the sidebar gives me a "subreddit not found - the subreddit redditcom couldn't be found." Notice it's missing the dot. 

Slide version: 5.6.5-alpha5.1
Android version: 7.1.1

**Comment by ccrama** [comment]
Slide automatically removes all non alphanumeric characters from subreddit names, I think this is the only subreddit with that and it would be a lot of work to add support for that subreddit. Don't know if it's going to be fixed

**Comment by incIined** [comment]
It's all cool, I'm not that lazy lol.

 But what about the first issue, where the app kinda freezes?

**Comment by Tensounder54** [comment]
Yeah there is somthing weird going on here. I think this should be marked as a `Feature Request`. Might toy around with getting the sub working on slide at some point if I get board again.
